app/views/orders/confirmation.html.erb

```diff
@@ -0,0 +1,22 @@
+<h1>Confirmation</h1>
+<!-- this is where the user can click yes or no and this can make the order complete -->
+<!-- or if the user decides they no longer want this order they can cancel -->
+<p> <%= link_to @order.id, order_path(@order.id) %></p>
+
+
+<% total = 0 %>
+<h3>Order Details</h3>
+<% @order.order_items.each do |order_item| %>
+  <% total+=order_item.quantity * order_item.product.price  %>
+  <p>Your order of: <%=link_to order_item.product.name, product_path(order_item.product.id) %><p> $<%= order_item.product.price %></p></p>
+
+<p>Name: <%=order_item.product.name %>
+<p>Quantity Ordered: <%= order_item.quantity %></p>
+<% end %>
+
+<p>Order placed on:  <%= @order.completed_time.localtime.strftime("%B %-d, %Y %I:%M %p") %>
+<p>Total Purchase: $ <%= total  %> <br/>
```

Comment:
The money is being stored as # of cents but the totals on this page are displayed as dollars. So it's actually displaying 100 x the actual cost. Check out the Rails currency helper method I mentioned in a different comment.